sunpy/data/sample.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+"""SunPy sample data files"""
+
+import sys as _sys
```

Comment:
instead of doing these imports as `_sys` what you might be able to do is : 

```python
__all__ = _sample_files.keys()
```
??